inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -278,6 +278,24 @@ const parseSettings = (settings: unknown) => {
 			})
 		}
 	}
+
+	const { sourceLanguageTag, languageTags } = settings as ProjectSettings
```

Comment:
why are you destructuring and then casting to an incorrect (!) type?

Just use `settings.sourceLanguageTag` and `setings.languageTags` directly?

Reply:
I did not know why settings are of type `unknown` – or is it true after we have checked for false with `.Check()` ?